### PR TITLE
notifications: fix layout

### PR DESCRIPTION
**Pull request by tylershuster**
fixes https://github.com/urbit/landscape/issues/218
fixes https://github.com/urbit/landscape/issues/237

before:
image

after:
image

also fixes an unfiled bug where leap crashes if state.suspendedfocus is invalid

I don't understand a lot of how the inbox is set up, but I think this is sane, so ping @liam-fitzgerald

### CODE REVIEW

**Comment by matildepark** [comment]
@vvisigoth did report that bug but no one could repro. He reported it by emailing me.

**Comment by matildepark** [comment]
Squeezing this in would be cool https://github.com/urbit/landscape/issues/148 (and see the others with that label)

**Comment by tylershuster** [comment]
- [ ] https://github.com/urbit/landscape/issues/171
- [ ] https://github.com/urbit/landscape/issues/170
- [ ] https://github.com/urbit/landscape/issues/148
- [ ] https://github.com/urbit/landscape/issues/146

**Comment by matildepark** [comment]
cc @urcades

**Comment by tylershuster** [comment]
(from urbit/landscape#171)
image

File: pkg/interface/src/views/apps/notifications/notification.tsx [hunk 1]
```diff
@@ -89,11 +88,22 @@ function NotificationWrapper(props: {
     return api.hark[func](notif);
   }, [notif, api, isMuted]);
 
+  const [visible, setVisible] = useState(false);
```
Comment (liam-fitzgerald) [inline pkg/interface/src/views/apps/notifications/notification.tsx:91]: pendanticism, but this is a good candidate for a custom hook. 
.e.g.
```typescript
const { hovering, bind } = useHovering();
// in component, bind is simply { onMouseEnter, onMouseLeave }
<Box {...bind}>

**Review by liam-fitzgerald — APPROVED** [review]
LGTM! minor nitpick about refactoring, else very good!

File: pkg/interface/src/views/apps/notifications/notification.tsx (hunk 1 above)
Comment (tylershuster) [reply pkg/interface/src/views/apps/notifications/notification.tsx]: Great idea; I wondered how I should do this. I've added another commit which adds and implements this.

**Comment by urcades** [comment]
these fixes look sane, whew — stamp!

**Pull request merged**